src/policy/protect_payload.rs

```diff
@@ -192,3 +229,31 @@ impl ForwardingRule {
         self
     }
 }
+
+// ———————————————————————————————— Hash primitive ———————————————————————————————— //
+
+fn hash_payload(size_to_hash: usize, pc_start: usize) -> [u8; 32] {
+    let payload_start: usize = 0x80400000;
+    let payload_end: usize = 0x80400000 + size_to_hash;
```

Comment:
Unfortunately this is not going to work on the board, the addresses are different.

The configuration file already has a PAYLAOD_START_ADDRESS field, could you use that instead? We have a `TARGET_PAYLOAD_ADDRESS` constant defined in `config.rs`.

Reply:
I would suggest to fix this in another commit properly with the rest of the protect payload policy